src/main/java/ExitCommand.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Deals with exiting the chat
+ */
+public class ExitCommand extends Command {
+
+    @Override
+    public boolean isExit(){
```

Comment:
nit: for method names try to use verbs instead (from textbook), maybe something like exit() will be better. Nouns are more used for variables/fields instead :)